PAWS/Onboarding/InvitationCodeView.swift

```diff
@@ -0,0 +1,156 @@
+//
+// This source file is part of the PAWS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Firebase
+import FirebaseAuth
+import FirebaseFunctions
+import SpeziOnboarding
+import SpeziValidation
+import SpeziViews
+import SwiftUI
+
+
+struct InvitationCodeView: View {
+    @Environment(OnboardingNavigationPath.self) private var onboardingNavigationPath
+    @State private var invitationCode = ""
+    @State private var viewState: ViewState = .idle
+    @ValidationState private var validation
+    
+    
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 32) {
+                invitationCodeHeader
+                Divider()
+                Grid(horizontalSpacing: 16, verticalSpacing: 16) {
+                    invitationCodeView
+                }
+                    .padding(.top, -8)
+                    .padding(.bottom, -12)
+                Divider()
+                OnboardingActionsView(
+                    primaryText: "Redeem Invitation Code",
+                    primaryAction: {
+                    guard validation.validateSubviews() else {
+                        return
+                    }
+                    
+                    await verifyOnboardingCode()
+                    },
+                    secondaryText: "I Already Have an Account"
+                ) {
+                    try Auth.auth().signOut()
+                    onboardingNavigationPath.nextStep()
+                }
```

Comment:
I would suggest a consistent parameter formatting and indentation here:
```suggestion
                    secondaryText: "I Already Have an Account",
                    secondaryAction: {
                        try Auth.auth().signOut()
                        onboardingNavigationPath.nextStep()
                    }
                )
```